evm/utils/blocks.py

```diff
@@ -26,12 +32,27 @@ def persist_block_to_db(db, block):
         rlp.encode(block.hash, sedes=rlp.sedes.binary),
     )
 
+
+def get_score(db, block_hash):
+    if block_hash == GENESIS_PARENT_HASH:
+        return GENESIS_DIFFICULTY
```

Comment:
Hmm, yeah, it could be but then we'd probably need to extend the persist_block_to_db() function to take the block's total score as well as it doesn't have access to the evm. Would it be ok if I leave it as a TODO for now?